consensus/hotstuff/model/block.go

```diff
@@ -69,10 +69,10 @@ func NewCertifiedBlock(block *Block, qc *flow.QuorumCertificate) (CertifiedBlock
 	return CertifiedBlock{Block: block, CertifyingQC: qc}, nil
 }
 
-// ID returns unique identifier for the block.
+// BlockID returns unique identifier for the block.
 // To avoid repeated computation, we use value from the QC.
```

Comment:
```suggestion
// BlockID returns a unique identifier for the block (the ID signed to produce a block vote).
// To avoid repeated computation, we use value from the QC.
// CAUTION: This is not a cryptographic commitment for the CertifiedBlock model.
```